storage/test/storage/SortedArrayStorageTest.java

```diff
@@ -0,0 +1,10 @@
+package storage;
+
+import storage.sorted.SortedArrayStorage;
+
+public class SortedArrayStorageTest extends ArrayStorageTest {
+
+    public SortedArrayStorageTest() {
+        super(new SortedArrayStorage(MAX_TEST_STORAGE_SIZE));
```

Comment:
There is no need in modifying the original storage size. There is no business case for such thing.

Reply:
Add one more constructor